website/package.json

```diff
@@ -8,13 +8,12 @@
     "swizzle": "docusaurus swizzle",
     "docusaurus": "docusaurus"
   },
-  "devDependencies": {},
   "dependencies": {
     "@docusaurus/core": "2.2.0",
+    "@docusaurus/plugin-client-redirects": "2.2.0",
```

Comment:
idk why but yarn decided to reorder these items when I ran `yarn run`, let's leave it like that because sooner or later somebody else will probably run into the same thing